src/test/java/pl/project13/maven/git/NativeAndJGitProviderTest.java

```diff
@@ -51,11 +51,20 @@
   public static final String DEFAULT_FORMAT_STRING  = "dd.MM.yyyy '@' HH:mm:ss z";
   public static final String ISO8601_FORMAT_STRING = "yyyy-MM-dd'T'HH:mm:ssZZ";
 
+// I feel there could be a mistake in the AvailableGitTestRepo's enum values
+// the MAVEN_GIT_COMMIT_ID_PLUGIN has the same directory as WITH_ONE_COMMIT_WITH_SPECIAL_CHARACTERS
+// I think the GIT_COMMIT_ID has the value which should be planned to be the plugin repo itself
+// If we use the GIT_COMMIT_ID repo this test will hang when nativeGit case selected in Windows definitely
```

Comment:
same comment as above? not sure this is needed, definitely should not stay in the codebase